src/index.js

```diff
@@ -491,7 +491,7 @@ class ReactNativeModal extends Component {
         <TouchableWithoutFeedback onPress={onBackdropPress}>
           <View
             ref={ref => (this.backdropRef = ref)}
-            useNativeDriver={useNativeDriver}
+            useNativeDriver={true}
             style={[
               styles.backdrop,
```

Comment:
Makes sense (the idea initially was to allow custom styling for it, hence it was not a default).
Thank you for the PR.